packages/metrics/src/features/metrics/constants.ts

```diff
@@ -8,13 +8,47 @@
  *
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
-export const OPENCRVS_SPECIFICATION_URL = 'http://opencrvs.org/specs/'
-export const CRUD_BIRTH_RATE_SEC = 'id/statistics-crude-birth-rates'
-export const JURISDICTION_TYPE_SEC = 'id/jurisdiction-type'
-export const TOTAL_POPULATION_SEC = 'id/statistics-total-populations'
-export const MALE_POPULATION_SEC = 'id/statistics-male-populations'
-export const FEMALE_POPULATION_SEC = 'id/statistics-female-populations'
-export const JURISDICTION_TYPE_IDENTIFIER = 'id/jurisdiction-type'
```

Comment:
Hey sorry, you can keep these values as constants. They are not going to be configured. Anything starting with process.env is more important

Reply:
In this file in metrics there's actually nothing to configure